packages/lib-classifier/src/components/Classifier/components/Layout/components/DefaultLayout/DefaultLayout.js

```diff
@@ -12,20 +12,20 @@ import QuickTalk from '@components/Classifier/components/QuickTalk'
 import SubjectViewer from '@components/Classifier/components/SubjectViewer'
 import TaskArea from '@components/Classifier/components/TaskArea'
 
-const subjectSize = '45.333rem'
-const taskSize = '25.333rem'
+const subjectSize = 'minmax(45.333rem, auto)'
```

Comment:
```suggestion
const largeSubjectSize = 'minmax(45.333rem, auto)'
const mediumSubjectSize = 'minmax(25.333rem, auto)'
```

Reply:
I think I’m missing something but this looks the same as:
```suggestion
const subjectSize = 'minmax(25.333rem, auto)'
```

Reply:
I've dismissed my suggestions - 'minmax(25.333rem, auto)' - would be the better way to implement, stinks that also had problems.

I think increase legibility of both. Per the mouse over in Chrome the zoom + and - buttons show "Make Text Larger" or "Make Text Smaller", which to me implies increasing both is ok/good, but I'm not sure.